lib/src/10_auth/auth_screen.dart

```diff
@@ -128,10 +151,38 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
     );
   }
 
-  Widget _buildCreateAccountScreen(BuildContext context) {
-    final TextEditingController emailController =
-        new TextEditingController(text: this._email);
+  Widget _buildTextBox(TextEditingController controller, FocusNode currentNode,
+      FocusNode nextNode, String fieldName, Future<Widget> Function() submit) {
+    
+    handleKey(RawKeyEvent key) {
+      if (!(key is RawKeyDownEvent)) {
```

Comment:
use operator 'is!' :

(key is! RawKeyDownEvent)

Reply:
done